student.rb

```diff
@@ -1,6 +1,8 @@
 require_relative './person'
 
 class Student < Person
+  attr_accessor :classroom
```

Comment:
- [ ] @sanieni6 Great work in your student class however the class should have only getters for attr_accessors since it's only being read and no changes made to it.